js/controller.js

```diff
@@ -0,0 +1,74 @@
+(function (window) {
+    'use strict';
+    function Controller(model, view){
+        console.log('controller created!');
+        this.model = model;
+        this.view = view;
+        var self = this;
+        this.view.bind('newTodo', function (title) {
+            self.addItem(title);
+        });
+        this.view.bind('itemRemove', function (item){
+            self.removeItem(item.id);
+        });
+        this.view.bind('itemEdit', function(item){
+            self.editItem(item.id);
+        });
+        this.view.bind('itemEditDone', function(item){
+            self.editItemSave(item.id, item.title);
+        });
+        this.showAll();
```

Comment:
```suggestion
        this.model = model;
        this.view = view;
        var self = this;
        this.view.bind('newTodo', function (title) {
            self.addItem(title);
        });
        this.view.bind('itemRemove', function (item){
            self.removeItem(item.id);
        });
        this.view.bind('itemEdit', function(item){
            self.editItem(item.id);
        });
        this.view.bind('itemEditDone', function(item){
            self.editItemSave(item.id, item.title);
        });
        this.showAll();
```
```suggestion
        var self = this;
        self.model = model;
        self.view = view;

        self.view.bind('newTodo', function (title) {
            self.addItem(title);
        });
        self.view.bind('itemRemove', function (item){
            self.removeItem(item.id);
        });
        self.view.bind('itemEdit', function(item){
            self.editItem(item.id);
        });
        self.view.bind('itemEditDone', function(item){
            self.editItemSave(item.id, item.title);
        });
        self.showAll();
```

self에 this를 가져왔는데, this와 self를 모두 사용하고 있어서 한 쪽으로 통일해주면 더 좋을거 같아요.